app/theme/brave/BRANDING

```diff
@@ -1,6 +1,6 @@
 COMPANY_FULLNAME=Brave Software, Inc.
 COMPANY_SHORTNAME=Brave Software
-PRODUCT_FULLNAME=Brave-Browser
+PRODUCT_FULLNAME=Brave 
```

Comment:
nit: trailing space

Reply:
I'm not sure if this will have extra consequences, but we want to be able to install Brave and Brave-Browser side by side.  cc @simonhong

Reply:
I think for this to be accepted, at least in part faster, pls split out the icon updates from the branding name changes.

Reply:
> I'm not sure if this will have extra consequences, but we want to be able to install Brave and Brave-Browser side by side.

This change makes stable version as `Brave` in launchpad.
If user has muon, two `Brave` will be shown in launch pad.

Reply:
Ah, I didn't take that into consideration. Could we compromise on just removing the hyphen but keeping Browser in these? My hope was just to omit the hyphens for the display names to make the app feel a little more polished. So this would be "Brave Browser", "Brave Browser Dev", etc...